examples/third-parties/codesandbox/tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "examples/**/*.ts",
     "scripts/**/*.ts",
     "src/**/*.ts",
-    "tests/**/*.spec.ts",
-  ],
```

Comment:
Please keep the trailing comma because it's our Wechaty org style.

Reply:
I saw that on CodeSandbox it is producing errors because of the trailing commas, that's why I removed them.

Reply:
Ok, got it.